plugins/plugin-markdown-image/tsconfig.build.json

```diff
@@ -2,7 +2,10 @@
   "extends": "../../tsconfig.build.json",
   "compilerOptions": {
     "rootDir": "./src",
-    "outDir": "./lib"
+    "outDir": "./lib",
+    "paths": {
+      "@vuepress/markdown": ["./src/types.d.ts"]
```

Comment:
It would be verbose to require this config. What about putting the `declare module` clause inside `options.ts`?